docs/csharp/whats-new/csharp-7.md

```diff
@@ -198,10 +198,9 @@ enables inspecting an object and its properties to determine if that object
 satisfies the sought pattern. You use the `when` keyword to specify additional
 rules to the pattern.
 
-### `is` expressions
+### `is` expression
 
-The `is` pattern expressions extend the familiar `is` operator
-to query a object beyond its type.
+The `is` expression extends the familiar `is` operator to query an object beyond its type.
```

Comment:
Nit: although the is expression is differentiated from the is operator here, it strikes me as useful to preserve the word "pattern" here, so that it reads, "The `is` pattern expression extends..." That more clearly differentiates it from the standard `is` operator.